src/App.vue

```diff
@@ -3,7 +3,8 @@
     <div id="nav">
       <router-link to="/">Home</router-link> |
       <router-link to="/about">About</router-link> |
-      <router-link to="/roma">Roma</router-link>
+      <router-link to="/recipeinformation">RecipeInformation</router-link> |
```

Comment:
Use [named routers](https://router.vuejs.org/guide/essentials/named-routes.html)
```suggestion
      <router-link :to="{ name: 'RecipeInformation', params: recipeId: 0 }">RecipeInformation</router-link> |
```